_posts/2015-07-25-tutorial-virtualbox.markdown

```diff
@@ -0,0 +1,235 @@
+
+De asemenea, Virtual Box nu este dependent de sistemul de operare, astfel că poate fi instalat și folosit pe orice platformă.
+
+**De ce să folosim Virtual Box?**
+
+În primul rând, cu Virtual Box putem avea mașini cu diferite configurații și sisteme de operare, ce pot fi pornite simultan. Acest lucru ne ajută foarte mult atunci când avem de testat comportamente ale aplicațiilor pe diferite platforme. Pe lângă aceasta, Virtual Box ne permite să salvăm stări ale mașinilor (Snapshot), ca mai apoi să revenim la ele, fără să se fi alterat dacă am făcut o modificare greșita între timp.
+
+## Instalare
+
+### Mac OS X
+
+Pentru a instala Virtual Box, accesați [pagina de download](https://www.virtualbox.org/wiki/Downloads) și descărcați versiunea pentru Windows, _VirtualBox 5.0 for OS X hosts_. Imaginea descărcată, după ce va fi descărcată și deschisă, va arăta ca în imaginea de mai jos:
+
+[<img alt="Descriere imagine" class="noshadow" src="/assets/images/virtualbox/macos/dmgcontent.png" />](/assets/images/virtualbox/macos/dmgcontent.png)
+
+Utilitarul de instalare se numește `VirtualBox.pkg`. Accesați utilitarul pentru a continua instalarea. Pașii de instalare sunt descriși în continuare:
+
+[<img alt="Verificare cerințe" class="noshadow" src="/assets/images/virtualbox/macos/oscheck.png" />](/assets/images/virtualbox/macos/oscheck.png)
+[<img alt="Meniu opțiuni" class="noshadow" src="/assets/images/virtualbox/macos/optsmenu.png" />](/assets/images/virtualbox/macos/optsmenu.png)
+
+În acest punct, două opțiuni de configurare sunt disponibile: **Customize** și **Change install location**. Prin apăsarea butonului *Customize* putem modifica ce componente vor fi instalate, precum ilustrează imaginea de mai jos:
+
+[<img alt="Customizare componente" class="noshadow" src="/assets/images/virtualbox/macos/custominstall.png" />](/assets/images/virtualbox/macos/custominstall.png)
+
+Prin apăsarea butonului *Change install location* putem modifica locația unde va fi instalat *VirtualBox*, precum ilustrează imaginea de mai jos:
+
+[<img alt="Customizare cale instalare" class="noshadow" src="/assets/images/virtualbox/macos/location.png" />](/assets/images/virtualbox/macos/location.png)
+
+După apăsarea butonului **Install**, utilizatorul va fi solicitat să își introducă parola pentru a putea continua instalarea:
+
+[<img alt="Solicitare parolă" class="noshadow" src="/assets/images/virtualbox/macos/password.png" />](/assets/images/virtualbox/macos/password.png)
+
+Din acest moment, instalarea va progresa fără a necesita orice interacțiune suplimetară din partea utilizatorului, proces ilustrat în imaginile următoare:
+
+[<img alt="Progres instalare" class="noshadow" src="/assets/images/virtualbox/macos/installprogress.png" />](/assets/images/virtualbox/macos/installprogress.png)
+[<img alt="Instalare completată" class="noshadow" src="/assets/images/virtualbox/macos/installsuccess.png" />](/assets/images/virtualbox/macos/installsuccess.png)
+
+### Windows
```

Comment:
Ar fi fain sa avem o consistenta pe partea de instalare intre Windows si OS X. Puteti descrie mai in detaliu pasii de instalare pe Windows, la fel cum e la OS X?